pkg/controllers/dynakube/connectioninfo/oneagent/conditions.go

```diff
@@ -0,0 +1,22 @@
+package oaconnectioninfo
```

Comment:
It's already a subfolder of `oneavent/connectioninfo`. By convention package name must be `conditions`.

Anyway you import it like `pkg/controllers/dynakube/connectioninfo/oneagent/conditions`

Reply:
this package was not named by me, and it is not a separate package 😅

Reply:
yeah I know :) but I dont like the name